src/views/SignTransactionLedger.vue

```diff
@@ -0,0 +1,408 @@
+
+            <div class="bottom-container blur-target" :class="{ 'full-height': state !== constructor.State.OVERVIEW }">
+                <LedgerUi ref="ledger-ui" small></LedgerUi>
+                <transition name="transition-fade">
+                    <Loader v-if="state !== constructor.State.OVERVIEW"
+                        :state="state === constructor.State.FINISHED ? 'success' : 'loading'"
+                        :title="loaderTitle">
+                    </Loader>
+                </transition>
+            </div>
+
+            <transition name="transition-fade">
+                <AccountDetails v-if="shownAccountDetails"
+                    :address="shownAccountDetails.address"
+                    :image="shownAccountDetails.image"
+                    :label="shownAccountDetails.label"
+                    :walletLabel="shownAccountDetails.walletLabel"
+                    :balance="shownAccountDetails.balance"
+                    @close="shownAccountDetails = null">
+                </AccountDetails>
+            </transition>
+        </SmallPage>
+
+        <button class="global-close nq-button-s" @click="_close"
+            :class="{ hidden: state !== constructor.State.OVERVIEW }">
+            <span class="nq-icon arrow-left"></span>
+            {{ request.kind === 'checkout' ? 'Cancel Payment' : `Back to ${request.appName}` }}
+        </button>
+        <Network ref="network" :visible="false"/>
+    </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Watch } from 'vue-property-decorator';
+import { Account, AccountDetails, PageBody, PageHeader, SmallPage } from '@nimiq/vue-components';
+import Network from '@/components/Network.vue';
+import LedgerApi from '../lib/LedgerApi';
+import LedgerUi from '../components/LedgerUi.vue';
+import Loader from '../components/Loader.vue';
+import { Static } from '../lib/StaticStore';
+import { Getter } from 'vuex-class';
+import { State as RpcState } from '@nimiq/rpc';
+import { ParsedCheckoutRequest, ParsedSignTransactionRequest, RequestType } from '../lib/RequestTypes';
+import { WalletInfo } from '../lib/WalletInfo';
+import { ERROR_CANCELED, TX_VALIDITY_WINDOW } from '../lib/Constants';
+import { Utf8Tools } from '@nimiq/utils';
+import Config from 'config';
+
+interface AccountDetailsData {
+    address: string;
+    label: string;
+    image?: string;
+    walletLabel?: string;
+    balance?: number;
+}
+
+@Component({components: {Account, PageBody, PageHeader, SmallPage, LedgerUi, Loader, AccountDetails, Network }})
+export default class SignTransactionLedger extends Vue {
+    private static readonly State = {
+        OVERVIEW: 'overview',
+        SENDING_TRANSACTION: 'sending-transaction',
+        FINISHED: 'finished',
+    };
+
+    @Static private rpcState!: RpcState;
+    @Static private request!: ParsedSignTransactionRequest | ParsedCheckoutRequest;
+    @Getter private findWalletByAddress!: (address: string) => WalletInfo | undefined;
```

Comment:
This function as a second mandatory parameter `includeContracts: boolean` as of #214.